readthedocs_build/config/config.py

```diff
@@ -78,6 +78,8 @@ class BuildConfig(dict):
     PYTHON_EXTRA_REQUIREMENTS_INVALID_MESSAGE = (
         '"python.extra_requirements" section must be a list.')
 
+    PYTHON_SUPPORTED_VERSIONS = [2, 2.7, 3, 3.3, 3.4, 3.5, 3.6]
```

Comment:
I think this can be agnostic to what versions we actually support, this is just checking for valid version numbers ATM. This decision should be left up to our implementation on RTD, i think.